tdrs-frontend/src/components/PrivateRoute/PrivateRoute.js

```diff
@@ -36,13 +37,16 @@ function PrivateRoute({ children, history, path }) {
 
   return authenticated ? (
     <Route path={path}>
```

Comment:
The main heading of the page and the title of the document are set by the private route. The title gets passed in from the router, to the private template, if the user is authorized.